Detections/W3CIISLog/ProxyShellPwn2Own.yml

```diff
@@ -0,0 +1,42 @@
+id: 968358d6-6af8-49bb-aaa4-187b3067fb95
+name: Exchange ProxyShell Pwn2Own
+description: |
+  'This query looks for suspicious request patterns to Exchange servers that fit patterns recently
+  blogged about by PeterJson.
+  Reference: https://peterjson.medium.com/reproducing-the-proxyshell-pwn2own-exploit-49743a4ea9a1'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AzureMonitor(IIS)
+    dataTypes:
+      - W3CIISLog
+queryFrequency: 1d
+queryPeriod: 14d
```

Comment:
This is a detection, I don't recommend we look back 14 days on a daily run.  Can you change this to 1d?  Or maybe even more often if we think this is High, could be 6 hours or less.

Reply:
Updated to run every 12 hours